### DISCUSSION TITLE
Can I deploy a keda scaled job using this library? If yes, can anyone give a short guide to me

### DISCUSSION THREAD

**Discussion by ranjithnktrrr** [kubernetes-client/csharp#1209] [General]
I was trying to deploy keda scaled job using this sdk.  Can anyone help on this whether it is possible using the sdk ? If yes it would be helpful for me if someone provide a code snippet

**Comment by tg123** [comment]
CRD https://github.com/kubernetes-client/csharp/tree/master/examples/customResource
